packages/audioplayers/lib/src/audioplayer.dart

```diff
@@ -36,6 +43,8 @@ class AudioPlayer {
     _playerState = state;
   }
 
+  StreamSubscription? _onPlayerCompleteStreamSubscription;
```

Comment:
Can't we declare it with `late`. It should always be available, as it's initialized in the constructor.
```suggestion
  late StreamSubscription _onPlayerCompleteStreamSubscription;
```